src/ecs/activation.ts

```diff
@@ -10,10 +12,21 @@ import { runCommandInContainer } from './commands/runCommandInContainer'
 import { EcsContainerNode } from './explorer/ecsContainerNode'
 import { EcsServiceNode } from './explorer/ecsServiceNode'
 import { ecsDocumentationUrl } from '../shared/constants'
+import { getLogger } from '../shared/logger'
 
 export async function activate(ctx: ExtContext): Promise<void> {
     ctx.extensionContext.subscriptions.push(
         vscode.commands.registerCommand('aws.ecs.runCommandInContainer', async (node: EcsContainerNode) => {
+            if (!(node instanceof EcsContainerNode)) {
+                getLogger().error('Cannot run command on node: %O', node)
+                vscode.window.showErrorMessage(
+                    localize(
+                        'AWS.explorerNode.notLoaded',
+                        'This resource may not be fully loaded yet. Please try again.'
+                    )
+                )
+                return
```

Comment:
Not a blocker at all, but it'd be interesting to see how frequently users run into this issue by emitting telemetry, probably as 'failed' with some extra metadata e.g. `reason: 'not loaded'`.